logging.tf

```diff
@@ -1,43 +1,43 @@
-resource "aws_s3_bucket" "bucket_logs" {
```

Comment:
Since there is no references to the node groups, or simphera instance resources, try uncommenting this file.